### ISSUE TITLE
[Bug]: Path Error

### ISSUE THREAD

**Issue by omar-jab** [laravel/vs-code-extension#135]
### Extension Version

0.1.14

### PHP Binary

Local PHP

### Operating System

Windows

### What happened?

I’m experiencing an issue when using a Windows account with spaces in the username (e.g., test - name). The extension throws an error whenever it tries to execute certain operations. It seems the issue arises because the username or path is not being handled correctly.
![image](https://github.com/user-attachments/assets/05fd61ed-1a26-498d-b918-4c5d9ce0e4c1)

[error] "c:\Users\[username]\" is not recognized as an internal or external command,
an executable program, or a batch file.

**Comment by mikebronner** [comment]
Possible duplicate of #19 ?

**Comment by omar-jab** [comment]
yup, i think it's the same problem, sorry i didn't see that there was already a report about it

**Comment by joetannenbaum** [comment]
Closing this in favor of #19, we'll track progress there.
